_includes/cloud-table/cloud-summary-table-pk.md

```diff
@@ -1,6 +1,6 @@
-The table ID column (`_id`) is used as the table primary key and must uniquely represent a record in your table.
+The `_id` column in each table represents the primary key for each column.
```

Comment:
primary key for the table, not the column

Reply:
fixed :D